run.sh

```diff
@@ -3,7 +3,7 @@
 if nvcc --version
 then
   export CUDA_VISIBLE_DEVICES=0
-  export LLAMA_N_CTX=44000
+  export LLAMA_N_CTX=43000
```

Comment:
why is that?

Reply:
no idea, vllm update also shrank the available ctx